packages/plugin-js-packages/src/lib/runner/index.ts

```diff
@@ -104,9 +116,10 @@ async function processAudit(
 
   const uniqueResults = pm.audit.postProcessResult?.(fulfilled) ?? fulfilled;
 
-  return supportedDepGroups.map(depGroup =>
+  return compatibleAuditDepGroups.map(depGroup =>
     auditResultToAuditOutput(
-      uniqueResults[depGroup],
+      // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
```

Comment:
Here only the results of dependency groups that were configured will appear. But TS is not smart enough to know which those are. Not sure if there is a way around this.

Reply:
I'll take that as a no 😅 okay.

Reply:
Sorry, yeah, can't think of a better way :sweat_smile: